### PR TITLE
Spelling: amount of newlines matches, characters

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/checks/chars.py [hunk 1]
```diff
@@ -327,10 +327,10 @@ class EscapedNewlineCountingCheck(CountingCheck):
 
 
 class NewLineCountCheck(CountingCheck):
-    r"""Check whether there is same amount of new lines"""
+    r"""Check whether the amount of newlines matches"""
     string = '\n'
     check_id = 'newline-count'
-    name = _('Newline charater count mismatch')
+    name = _('Mismatching newline character count')
```
Comment (nijel) [inline weblate/checks/chars.py]: I was thinking about calling this rather "Mismatching lines count". What do you think?

Comment (comradekingu) [reply weblate/checks/chars.py]: It is a bit hard to decipher. "Mismatching line-count" would do.
Is it always technically correct vs. number of newline chars?

Comment (nijel) [reply weblate/checks/chars.py]: It just counts newline chars. When translating to Czech I realized that it's easier to understand when talking about number of lines than number of newline chars...

Comment (comradekingu) [inline weblate/checks/chars.py]:
```suggestion
    name = _('Mismatching line breaks')
```
I think this (typographic) variant makes sense over "Mismatching number of new lines".
"Added line breaks" does away with the idea of wrapping, but alas.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**